_posts/2017-04-04-what-every-developer-should-know-about-ir.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: post
+title: What every developer should know about Information Retrieval
+author: juhofriman
+excerpt: Effective and intuitive search functionalities are crucial part of nearly every system. This blog post tries to introduce reader some of the science behind information retrieval (IR) as it is vastly studied since the sixties, yet the field is not too well known within the developer community.
+tags:
+- Information Retrieval
+- Elasticsearch
+- Postgresql
+- Querying
+- Indexing
+- Natural language processing
+---
+
+Effective and intuitive search functionalities are crucial part of nearly every system, but my gut feeling is that search functionalities is usually implemented in an ad hoc manner - just creating something that returns results to user and let the user to tinker with search and just try to cope with it. This blog post tries to introduce reader some of the science behind information retrieval (IR) as it is vastly studied since the sixties, yet the field is not too well known within the developer community. A suggested reading list is collected at the end.
```

Comment:
are _a_ crucial part
that _the_ search functionalities _are_
returns results to user and _lets_